### PR TITLE
Adding Azure computer vision tool

### PR DESCRIPTION
**Pull request by ajhofmann**
Azure computer vision tool for OCR and image classification

### CODE REVIEW

File: llama_hub/tools/azure_cv/base.py [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+
+from llama_index.tools.tool_spec.base import BaseToolSpec
+from typing import Optional, List
+import requests
+import urllib.parse
+
+CV_URL_TMPL = "https://{resource}.cognitiveservices.azure.com/computervision/imageanalysis:analyze"
+
+class AzureCVToolSpec(BaseToolSpec):
+    """Wolfram Alpha tool spec."""
```
Comment (jerryjliu) [inline llama_hub/tools/azure_cv/base.py:11]: docstring

File: llama_hub/tools/azure_cv/README.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+You will need to set up an api key and computer vision instance using Azure, learn more here: https://azure.microsoft.com/en-ca/products/cognitive-services/computer-vision
+
+## Usage
+
+This tool has more extensive example usage documented in a Jupyter notebook [here](https://github.com/emptycrown/llama-hub/tree/main/llama_hub/tools/notebooks/azure_vision.ipynb)
+
+Here's an example usage of the AzureCVToolSpec.
+
+```python
+from llama_hub.tools.google_search.base import AzureCVToolSpec
```
Comment (jerryjliu) [inline llama_hub/tools/azure_cv/README.md:14]: import is wrong

**Comment by ajhofmann** [comment]
Once a make a few more of these Azure tools I'll make a more advanced combined notebook playing around with some image and audio stuff!

**Comment by jerryjliu** [comment]
> Once a make a few more of these Azure tools I'll make a more advanced combined notebook playing around with some image and audio stuff!

awesome!

**Review by jerryjliu — APPROVED** [review]
@ajhofmann would you be able to help fix merge conflicts?

**Comment by ajhofmann** [comment]
> @ajhofmann would you be able to help fix merge conflicts?

Should be fixed but worried the two Azure PRs are gonna create another one, I'll fix it up again if they do  👌

**Comment by ajhofmann** [comment]
Combined all the Azure tools into one PR for easy of merging :ok: https://github.com/emptycrown/llama-hub/pull/459/files

**Pull request closed without merging**